### ISSUE TITLE
Annealing beta in the loss function

### ISSUE THREAD

**Issue by sgalkina** [masa-su/pixyz#87]
Hi! Thanks again for the library.

Is it possible to implement KL cost annealing in the loss function of Model object? The full loss function is assigned when the Model object is created, thus not allowing to change the beta term during the training.

**Comment by masa-su** [comment]
@sgalkina 
In Pixyz, the [`Parameter`](https://docs.pixyz.io/en/latest/losses.html#parameter) class enables you to set the parameter in the loss function as the input variable. So, if you set the coefficient of the KL term in the loss function as this class, you can change this value during training.

![スクリーンショット 2019-10-15 16 41 35](https://user-images.githubusercontent.com/11865486/66810673-ac6e6a80-ef6a-11e9-8b3d-490611150e34.png)

**Comment by sgalkina** [comment]
@masa-su thanks a lot!
